### PR TITLE
Preprod: movie: Signs (2002) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
Rating 4/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 1]
```diff
@@ -2467,6 +2467,58 @@ computer hacking [Digicorp tried to "jack in" to the Sunway Systems computer sys
...
+2002-08-02
+
+:: References
+https://en.wikipedia.org/wiki/Signs_(2002_film)
+
+:: Ratings
+4 <paul>
+
+:: Major Themes
+extraterrestrial being [The story follows Graham and his family as they barricade themselves in their isolated farmhouse in the midst of an alien invasion of Earth.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:2495]: Off topic motivation.

Comment (odinlake) [reply notes/stories/film/film-scifi-2000s.st.txt:2495]: Maybe I'm wrong but I think you didn't use this theme in Impostor. Yet there was an alien invasion in both cases.

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 2]
```diff
@@ -2467,6 +2467,58 @@ computer hacking [Digicorp tried to "jack in" to the Sunway Systems computer sys
...
+:: Minor Themes
+human health condition [Morgan suffered from asthma and had a bad attack on night.]
+coping with personal failure [The former minor league baseball player Merrill was evidently a bit torn up inside over never having made it to the majors.]
+remorse [Ray expressed regret for having caused Colleen's death and Graham's subsequent losing of his faith.]
+law enforcement [The local police officer Caroline Paski visited the Hess house a couple of times to investigate the strange goings on there.]
+coping with the death of a parent [The Hess children were coming to terms with that their mother had died and was never coming back.]
+hoaxes in society [A man on television stated that the crop circles had been dismissed as hoaxes in the 1980s.]
+speculative spaceship [The television news showed alien spacecrafts in the nigh sky above Mexico City.]
+the game of baseball [A local man showered Merrill with praise for for having hit a record breaking 507 foot home run in the minor leagues.]
+nuclear weapons [Morgan read in his UFO book that invading aliens would use ground tactics and hand-to-hand combat to prevent humans from nuking Earth, making the planet useless to them.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:2517]: That made zero sense to me.

**Review by odinlake — APPROVED** [review]
Minor comments inline but do as you please.

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 2 above)
Comment (odinlake) [reply notes/stories/film/film-scifi-2000s.st.txt:2517]: (perhaps it was meant not to?)

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt:2495]: Happily reworked the motivation. But we have *extraterrestrial being* as major in Impostor (2002).

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt]: I simplified the motivation, but the whole scenario as described in the film made no sense at all, lol.

**Pull request merged**